charts/spark-operator-chart/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: spark-operator
 description: A Helm chart for Spark on Kubernetes operator
-version: 1.2.1
+version: 1.2.2
```

Comment:
Could you bump up the chart version to 1.2.3? Another PR earlier that yours just took 1.2.2

Reply:
Done, had to force push since I forgot the git --signoff again